### DISCUSSION TITLE
Shortcuts

### DISCUSSION THREAD

**Discussion by funnie-monkey** [FreeTubeApp/FreeTube#4273] [General]
Is there anyway to customize or see the available shortcuts? I don't know if I'm just missing it or if it's something that should be a request. I'm mainly looking for a shortcut to clear and focus on the search bar instead of always having to use my mouse.

Thanks in advance.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @cgutierrez21 u can find the list of all our keyboard shortcuts here:

https://docs.freetubeapp.io/usage/keyboard-shortcuts/

Shortcuts are not customisable see open issue https://github.com/FreeTubeApp/FreeTube/issues/251

**Reply by funnie-monkey** [reply]
Sorry for the super late response but this is exactly what I was looking for(I don't know how I missed it 😅) thank you so much!!!
